### PR TITLE
Feature/SSLintegration

### PR DESCRIPTION
**Pull request by romulocintra**
Work in progress for integration of SSL.  #6

TODO : 

- [x] Generate SSL and server using site https
- [x] Add watch mode for ts files on dev
- [x] Add SSL to livereload
- [x] Add configuration to ssl true/false using devServer inside App 
- [x] fix unit test
- [x] add test to https
- [x] Review interfaces and cleanup code

### CODE REVIEW

**Comment by danbucholtz** [comment]
Now this is a legit PR! Nice work! 🎉

File: src/index.ts [hunk 1]
```diff
@@ -117,7 +130,7 @@ export async function run(argv: string[]) {
     });
   }
 
-  process.once('SIGINT', async () => {
+  process.once('SIGINT', async () => {;
```
Comment (jthoms1) [inline src/index.ts]: Looks like an unintended typo.

**Review by jthoms1 — COMMENTED** [review]
One issue that I have experienced which is more so with devcert than your PR is that when the cert does not exist and it needs to be created then their is a password prompt without any context as to why the password is being requested.  I could see this causing some confusion.  It appears that devcert does not give you the ability to ask if it had been created yet.

We might let this pass for now but most likely we will open a PR for devcert to allow us to check if it had been created so that we can write a message to the console to provide the user with context about why we are asking for the password.

**Comment by jthoms1** [comment]
I also opened a PR for devcert to expose their types that are generated.  https://github.com/davewasmer/devcert/pull/12

File: src/index.ts (hunk 1 above)
Comment (romulocintra) [reply src/index.ts]: Fixed thks

Comment (romulocintra) [reply src/index.ts]: By the away i will check if is posible to do something with prompt

**Comment by jthoms1** [comment]
Looks good! Thank you for your work, very clean PR!

**Pull request merged**